src/stories/3-fields/contact-number/contact-number.stories.tsx

```diff
@@ -50,6 +38,7 @@ export default {
 			description: "Specifies a default country for the input field",
 			table: {
 				type: {
+					// NOTE: change to explicit??
```

Comment:
actually you can change the controls to `select` and provide `InternationalCallingCodeMap` as the options